features/step_definitions/project.rb

```diff
@@ -193,7 +193,7 @@
     | project_name  | #{project_name}       |
     | node_selector | #{project_name}=label |
     })
-  step %Q/I store the ready and schedulable workers in the :nodes clipboard/
+  step %Q/I store the schedulable workers without taints in the :nodes clipboard/
```

Comment:
@ropatil010 FYI, now that you have added default value, you can simply call the step like 
```
step %Q/I store the schedulable workers without taints in the clipboard/
```